### PR TITLE
Conversion host authorization and use the deploy/delete playbooks in CI

### PR DESCRIPTION
**Pull request by jistr**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -66,13 +66,16 @@ test-func: reinstall
 		-e @$(ROOT_DIR)/tests/auth.yml \
 		$(FUNC_TEST_ARGS) test_all.yml
 
+# FIXME: remove the parameters from here, put them in a file (like we
```
Comment (ccamacho) [inline Makefile:69]: I pushed this:
https://github.com/os-migrate/os-migrate/pull/212

To adjust the parameters based on the scenario we will like to run.

File: Makefile [hunk 2]
```diff
@@ -75,22 +78,16 @@ test-e2e: reinstall
...
-		-e os_migrate_src_conversion_host_name=osm_uch_src \
-		-e os_migrate_dst_conversion_host_name=osm_uch_dst \
-		-e os_migrate_src_conversion_host_flavor=m1.medium \
-		-e os_migrate_dst_conversion_host_flavor=m1.medium \
-		-e os_migrate_src_client_key=~/ssh-ci/id_rsa.pub \
-		-e os_migrate_dst_client_key=~/ssh-ci/id_rsa.pub \
-		-e os_migrate_conversion_host_key=~/ssh-ci/id_rsa \
-		-e os_migrate_conversion_host_image=rhel-osp-migration-conversion-host.qcow2 \
+		-e os_migrate_conversion_external_net_name=public \
+		-e os_migrate_conversion_flavor_name=m1.medium \
```
[the remark this answers is not in the document]
Comment (jistr) [reply Makefile:88]: Yea what i have here is incorrect design. The Q is do we want to fix it right here or later? I'm fine with either. I see you assigned the Jira card for this to yourself, so would you be interested in doing the fix?

[the remark this answers is not in the document]
Comment (ccamacho) [reply Makefile:88]: Yes, I approved the pr because having it merged is more important than the fix. I'll push a variables fix for it.

**Comment by ccamacho** [comment]
Ship it!

**Pull request merged**